openfisca_uk/parameters/poverty/absolute_poverty_threshold_ahc.yaml

```diff
@@ -1,8 +1,12 @@
 description: Absolute poverty threshold for equivalised household net income, after housing costs.
 values:
-  2015-06-01:
-    value: 270
+  2010-01-01:
```

Comment:
should this be 2010-04-01, or does it have to align with the CPI dates?

Reply:
Yeah, the way the uprating is implemented, it won't make a difference unless we interpolate CPI further